### PR TITLE
Fixes issue #60, which is randomly caused by system file locking virus scanners

### PR DESCRIPTION
**Pull request by t-johnson**
Fixes issue #60, which is randomly caused by system file locking virus scann activity.
Don't delete and re-create directories - just clean the directory if it already exists

### CODE REVIEW

**Comment by FortuneN** [comment]
Thanks @t-johnson 
@tonyhallett What do you think?

**Review by tonyhallett — COMMENTED** [review]
I would also move the common EnsureEmptyDirectory method call out of the two util classes and into the FCCEngine.ReloadCoverage method.

```
.Select(p => p.Step("Run Coverage Tool", project =>
			{
                                // add here **************
                                FileUtil.EnsureEmptyDirectory(project.CoverageOutputFolder);


				// run the appropriate cover tool

				if (project.IsDotNetSdkStyle)
				{
					CoverletUtil.RunCoverlet(project, true);
				}
				else
				{
					OpenCoverUtil.RunOpenCover(project, true);
				}
			}))
```

File: FineCodeCoverage/Core/Coverlet/CoverletUtil.cs [hunk 1]
```diff
@@ -154,17 +154,7 @@ public static bool RunCoverlet(CoverageProject project, bool throwError = false)
...
-				File.Delete(project.CoverageOutputFile);
-			}
-
-			if (Directory.Exists(project.CoverageOutputFolder))
-			{
-				Directory.Delete(project.CoverageOutputFolder, true);
-			}
-
-			Directory.CreateDirectory(project.CoverageOutputFolder);
+			FineCodeCoverage.Engine.Utilities.FileUtil.EnsureEmptyDirectory(project.CoverageOutputFile);
```
[the remark this answers is not in the document]
Comment (t-johnson) [reply FineCodeCoverage/Core/Coverlet/CoverletUtil.cs:157]: Thanks, i'll make the changes.  i only tested with a .net framework project, so i didn't catch that last copy/paste error!

**Review by tonyhallett — APPROVED** [review]
Looks good to me. Thanks

**Pull request merged**